From 21178a0215107c1e06234d512c32ba96af6e2853 Mon Sep 17 00:00:00 2001
From: Mazhar <mazharx.sameullah@intel.com>
Date: Tue, 11 Mar 2014 12:33:58 +0530
Subject: [PATCH] Avoiding ANR due to Widevine Sniff taking long time to return

BZ: 174969

This fix will avoid ANR because of SniffWVM function taking
long time to return.

Change-Id: I4fcfe3b61c13b4c4d86f756bcc16ca1d111cebb9
Category: device enablement
Domain: AUDIO_MEDIA-Audio Framework
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: Mazhar <mazharx.sameullah@intel.com>
---
 media/libstagefright/DataSource.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/media/libstagefright/DataSource.cpp b/media/libstagefright/DataSource.cpp
index f4f15f7..dc08290 100644
--- a/media/libstagefright/DataSource.cpp
+++ b/media/libstagefright/DataSource.cpp
@@ -172,11 +172,12 @@ void DataSource::RegisterDefaultSniffers() {
     registerSniffer_l(SniffMP3);
     registerSniffer_l(SniffAAC);
     registerSniffer_l(SniffMPEG2PS);
-    registerSniffer_l(SniffWVM);
     registerSniffer_l(SniffAVI);
 #ifdef USE_INTEL_ASF_EXTRACTOR
     registerSniffer_l(SniffAsf);
 #endif
+    registerSniffer_l(SniffWVM);
+
 #ifdef ENABLE_AV_ENHANCEMENTS
     registerSniffer_l(ExtendedExtractor::Sniff);
 #endif
-- 
2.5.0

